Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -112,7 +113,7 @@ public sealed partial class HumanoidProfileEditor : Control
             IEntityManager entityManager, IConfigurationManager configurationManager)
         {
             RobustXamlLoader.Load(this);
-            _random = IoCManager.Resolve<IRobustRandom>();
+            IoCManager.Resolve<IRobustRandom>();
```

Comment:
huh

Reply:
buh rider told me it was unused and I used its refactor